tests/resource/resource_result_samples/extract_resource_result_precheck_error.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="IBM-1047"?>
+<securityresult xmlns="http://www.ibm.com/systems/zos/saf/IRRSMO00Result1">
+  <resource name="IRR.IRRSMO00.PRECHECK" class="XFACILIT" operation="listdata" requestid="ResourceRequest">
+    <command>
+      <safreturncode>8</safreturncode>
+      <returncode>16</returncode>
+      <reasoncode>4</reasoncode>
+      <image>RLIST   XFACILIT             (IRR.IRRSMO00.PRECHECK) </image>
+      <message>ICH13003I TESTING NOT FOUND</message>
```

Comment:
Is this the actual message that this returns?

Reply:
No, it should have been `<message>ICH13003I IRR.IRRSMO00.PRECHECK NOT FOUND</message>` and has now been changed.